torch/ao/quantization/fx/_model_report/detector.py

```diff
@@ -931,26 +932,55 @@ class OutlierDetector(DetectorBase):
         if p_r is above some threshold, then we consider the activations to have significant outliers
 
     Args:
-        ratio_threshold (float): The threshold for p_r to determine if there are outliers in activations
-            Should be between 0 and 1 (both non-inclusive)
+        ratio_threshold (float, optional): The threshold for p_r to determine if there are outliers in activations
+            Default: 3.5
         reference_percentile (float, optional): The denominator to find the relative scale of the 100th percentile
+            Should be between 0 and 1
+            Default: 0.975
```

Comment:
Also is there a good `ratio_threshold` for each `reference_percentile`, assuming the data follows a normal distribution? If so I think this would make it easier to provide a more appropriate default for `ratio_threshold` (as opposed to just 3.5) for different `reference_percentile` values

Reply:
https://www.internalfb.com/intern/anp/view/?id=2169566 there was work done in this notebook to determine some good values and their respective thresholds. These were picked based on the experiment done in that notebook as to what the ratio would look like in a normal distribution.

I change it so that the user can select something from a fixed set of values, and the values have already had their respective threshold value calculated.

Reply:
I also think I still want it to be a value the user can pass in, because depending on their own data distribution, what counts as an outlier or what percentile they want to look at can change drastically. Furthermore, because the ratio_threshold was set based on a normal distribution, if their data is inherently more of a two-peaks distribution, then this just wouldn't really work, which is why i thought allowing as much flexibility as possible was appropriate.

However, I see your point and I definitely think this is something to revisit later if we find that almost everything we encounter will just be a normal distribution and it will be better to just give users the different options to choose from there.